projects/packages/videopress/src/videopress-divi/class-videopress-divi-extension.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Divi integration for Videopress.
+ *
+ * @package automattic/jetpack-videopress
+ **/
+
+use Automattic\Jetpack\Assets;
+
+/**
+ * Divi extension.
+ **/
+class VideoPress_Divi_Extension extends DiviExtension {
+
+	/**
+	 * The extension's js file namespace.
+	 *
+	 * @var string
+	 */
+	const JETPACK_VIDEOPRESS_DIVI_PKG_NAMESPACE = 'jetpack-videopress-divi-pkg';
+
+	/**
+	 * The gettext domain for the extension's translations.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+	public $gettext_domain = 'jetpack-videopress-pkg';
+
+	/**
+	 * The extension's WP Plugin name.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+	public $name = 'videopress-divi';
+
+	/**
+	 * The extension's version
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+	public $version = '1.0.0';
+
+	/**
+	 * The VideoPress_Divi_Module.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var VideoPress_Divi_Module
+	 */
+	private $videopress_divi_module;
+
+	/**
+	 * The constructor.
+	 *
+	 * @param string $name The name.
+	 * @param array  $args The args.
+	 */
+	public function __construct( $name = 'videopress-divi', $args = array() ) {
+		$this->plugin_dir     = plugin_dir_path( __FILE__ );
+		$this->plugin_dir_url = plugin_dir_url( $this->plugin_dir );
+
+		parent::__construct( $name, $args );
+	}
+
+	/**
+	 * Executes when the ET builder module is loaded.
+	 *
+	 * @override
+	 */
+	public function hook_et_builder_modules_loaded() {
+		$this->hook_et_builder_ready();
+	}
+
+	/**
+	 * Loads custom modules when the builder is ready.
+	 *
+	 * @override
+	 */
+	public function hook_et_builder_ready() {
+		require_once plugin_dir_path( __FILE__ ) . 'class-videopress-divi-module.php';
```

Comment:
Same as before, I was wondering if we needed the require here?

Reply:
Unfortunately, this is also subclassing a Divi class